http-client/src/main/java/no/nav/familie/http/ecb/ECBRestClient.kt

```diff
@@ -0,0 +1,54 @@
+package no.nav.familie.http.ecb
+
+import no.nav.familie.http.client.AbstractRestClient
+import no.nav.familie.http.config.ECBRestTemplate
+import no.nav.familie.http.ecb.domene.ECBExchangeRatesData
+import no.nav.familie.http.ecb.domene.ExchangeRate
+import no.nav.familie.http.ecb.domene.toExchangeRates
+import org.springframework.beans.factory.annotation.Qualifier
+import org.springframework.context.annotation.Import
+import org.springframework.stereotype.Component
+import org.springframework.web.client.RestOperations
+import java.net.URI
+import java.time.LocalDate
+
+@Component
+@Import(ECBRestTemplate::class)
+class ECBRestClient(@Qualifier("ecbRestTemplate") private val restOperations: RestOperations) : AbstractRestClient(restOperations, "ecb") {
+
+    private final val ECBApiUrl = "https://sdw-wsrest.ecb.europa.eu/service/data/EXR/"
+
+    /**
+     * Henter valutakurser fra ECB (European Central Bank) for *currencies*
+     * @param frequency spesifiserer om valutakurser skal hentes for spesifikk dag eller for måned.
+     * @param currencies liste over valutakurser som skal hentes.
+     * @param exchangeRateDate dato man ønsker valutakurser for. Dersom *frequency* er MONTHLY hentes forrige måneds kurs, med mindre man spør med *exchangeRateDate* = siste dag i mnd.
+     * @return Liste over valutakurser med tilhørende kode, kurs og dato.
+     */
+    fun getExchangeRates(frequency: Frequency, currencies: List<String>, exchangeRateDate: LocalDate): List<ExchangeRate> {
+        val uri = URI.create("${ECBApiUrl}${frequency.toFrequencyParam()}.${toCurrencyParams(currencies)}.EUR.SP00.A/${frequency.toQueryParams(exchangeRateDate)}")
+        try {
+            return getForEntity<ECBExchangeRatesData>(uri).toExchangeRates()
+        } catch (e: Exception) {
```

Comment:
Her fanger vi opp alle exceptions som kan oppstå, men kaster en `"ECBClientException: Fant ingen valutakurser for $currencies på dato: $exchangeRateDate fra European Central Bank"`.

Kan vi havne i en situasjon der det faktisk finnes valutakurser, men det er noe annet som feiler?
Hvis så, så er det kanskje gunstig å catche spesifikke exceptions for å gi en mer spesifiserende feilmelding

Reply:
Sia vi sender med den originale exception-en i throw-kallet, blir heile feilmeldinga med i stacktracen, så eg er usikker på om det er nødvendig å vera meir spesifikk. Men kanskje ein tekst a la "Klarte ikke å hente valutakurser for...", som dekkjer det meste?

Reply:
Japp, jeg tror det skal dekke de fleste behovene hvis vi bytter til `Klarte ikke å hente valutakurser for..."` som du foreslår 🙌

Reply:
Ja, det burde være mulig! Ser litt på det 👀 👍

Reply:
La på litt ekstra håndtering for å ha litt forskjellige meldinger ut ifra hva slags feil det er.

Reply:
Ser 💯 ut!